h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -2004,6 +2004,98 @@ Examples:
+
+To compute Gains/Lift, H2O applies the model to the original dataset to find the response probability. The data is divided into groups by quantile thresholds of the response probability. The default number of groups is 16; if there are fewer than sixteen unique probability values, then the number of groups is reduced to the number of unique quantile thresholds. 
+
+**An example**: a response model predicts who will respond to a marketing campaign. If you have a response model, you can make more detailed predictions. You use the response model to assign a score to all 100000 customers and predict the results of contacting only the top 10000 customers, the top 20000 customers, and so on. You do this by:
+
+- taking the dataset, sending it through your model, and obtaining a list of predicted output which is the probability of positive response;
+- sorting your dataset according to the output of your model which is the probability of positive response (this probability can also be called the **score**) from highest to lowest;
+    
+    - In this case, the first bin contains the top 10000 customers with the highest response probability, the second bin contains the next 10000 customers with the highest response probability, and so on.
+
+**Cumulative gains and lift charts** are a graphical representation of the advantage of using a predictive model to choose which customers to target/contact. On the cumulative gains chart, the y-axis shows the percentage of positive responses out of a total possible positive responses. The x-axis shows the percentage of customers contacted. The lift chart shows how much more likely you are to receive responses than if you contacted a random sample of customers.
+
+Example:
+
+.. tabs::
+    .. code-tab:: r R
+
+        # Import the airlines dataset:
+        airlines <- h2o.importFile("https://s3.amazonaws.com/h2o-public-test-data/smalldata/testng/airlines_train.csv")
+
+        # Build and train the model:
+         model <- h2o.gbm(x = c("Origin", "Distance"), 
+                          y = "IsDepDelayed", 
+                          training_frame = airlines, 
+                          ntrees = 1, 
+                          gainslift_bins = 20)
+
+         # Plot the Gains/Lift chart:
+         gain_table = model@model$training_metrics@metrics$gains_lift_table
+         plot(gain_table$cumulative_data_fraction,
+              gain_table$cumulative_capture_rate,'l')
+
+.. figure:: images/gainslift_plot.png
```

Comment:
Plot needs to be updated with image like this (or you can use this one):

<img width="505" alt="Screen Shot 2021-11-11 at 1 05 14 PM" src="https://user-images.githubusercontent.com/4073698/141368911-b0365a8f-fad5-48af-bbe3-b6fce0a1c300.png">